docs/dev/README.md

```diff
@@ -242,6 +242,9 @@ There are three sets of people with extra permissions:
   They also have direct commit access, but all changes should via bors queue.
   It's ok to self-approve if you think you know what you are doing!
   bors should automatically sync the permissions.
+  Feel free to assign any PR to a reviewer with the relevant expertise to bring the work to their attention.
```

Comment:
```suggestion
  Feel free to request a review or assign any PR to a reviewer with the relevant expertise to bring the work to their attention.
```

Other contributors won't be able to assign reviewers, but some of them already have an idea about who should take a look at the code.